video_xblock/backends/brightcove.py

```diff
@@ -337,8 +337,9 @@ def basic_fields(self):
         return super(BrightcovePlayer, self).basic_fields + ('account_id',)
 
     advanced_fields = (
-        'player_id', 'start_time', 'end_time', 'handout', 'transcripts',
-        'download_transcript_allowed', 'token', 'default_transcripts'
+        'player_id', 'start_time', 'end_time', 'handout', 'playmedia_apikey',
+        'playmedia_file_id', 'transcripts', 'download_transcript_allowed',
+        'token', 'default_transcripts'
```

Comment:
Rather than extending `advanced_fields` in every backend, why not change it once in `VideoXBlock`?